.changeset/silver-dodos-turn.md

```diff
@@ -0,0 +1,5 @@
+---
+'@cloudfour/patterns': patch
+---
+
+Update Author component story code example
```

Comment:
I don't think a changeset is necessary because this does not affect the published package, so no version release is necessary